src/modules/common/components/fallback-screen/fallback-screen.component.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC, ReactNode } from 'react'
```

Comment:
```suggestion
import { FC, ReactNode } from 'react'
```
we don't need to import the whole module